fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/expression/StringExpNodeVisitor.java

```diff
@@ -0,0 +1,406 @@
+    public String exists(String selectString) {
+        StringBuilder result = new StringBuilder();
+        result.append("EXISTS (");
+        result.append(selectString);
+        result.append(")");
+        return result.toString();
+    }
+
+    @Override
+    public String notExists(String selectString) {
+        StringBuilder result = new StringBuilder();
+        result.append("NOT EXISTS (");
+        result.append(selectString);
+        result.append(")");
+        return result.toString();
+    }
+
+    @Override
+    public String eq(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        result.append(" = ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String neq(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        result.append(" != ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String gt(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        result.append(" > ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String gte(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        result.append(" >= ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String lt(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        result.append(" < ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String lte(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        result.append(" <= ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String literal(String value) {
+        StringBuilder result = new StringBuilder();
+        result.append("'");
+        result.append(value);
+        result.append("'");
+        return result.toString();
+    }
+
+    @Override
+    public String literal(Long value) {
+        StringBuilder result = new StringBuilder();
+        result.append(value);
+        return result.toString();
+    }
+
+    @Override
+    public String literal(Double value) {
+        StringBuilder result = new StringBuilder();
+        result.append(value);
+        return result.toString();
+    }
+
+    @Override
+    public String add(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        result.append(" + ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String sub(String left, String right) {
```

Comment:
would prefer to see this spelled out for readability ('subtract' vs. 'sub'). Same for mult and div.

Reply:
ok